src/serialisation.rs

```diff
@@ -15,45 +15,44 @@
 // Please review the Licences for the specific language governing permissions and limitations
 // relating to use of the SAFE Network Software.
 
-use cbor::{Encoder, Decoder};
+use bincode::SizeLimit;
+use bincode::rustc_serialize::{decode, DecodingError, encode, EncodingError};
 use rustc_serialize::{Encodable, Decodable};
 
 quick_error! {
-    /// Wrapper for Serialisation Errors. This is present because cbor code paths don't always return a
-    /// Result - they return an Option too.
+    /// Serialisation error.
     #[derive(Debug)]
     pub enum SerialisationError {
-        /// Encapsulated Cbor Error
-        CborError(err: ::cbor::CborError) {
-            description("Cbor error")
-            display("Cbor error: {}", err)
+        /// Error during serialisation (encoding).
+        SerialiseError(err: EncodingError) {
```

Comment:
Didn't know Clippy doesn't like that, but sure, why not. I'm anticipating more changes to this module anyway, as I'm porting everything to bincode. So don't merge yet.